tensorflow_lite_support/python/task/processor/proto/classification_options_pb2.py

```diff
@@ -13,6 +13,74 @@
 # limitations under the License.
 """Classification options protobuf."""
 
+import dataclasses
+from typing import Any, List, Optional
+
+from tensorflow.tools.docs import doc_controls
 from tensorflow_lite_support.cc.task.processor.proto import classification_options_pb2
 
-ClassificationOptions = classification_options_pb2.ClassificationOptions
+_ClassificationOptions = classification_options_pb2.ClassificationOptions
+
+
+@dataclasses.dataclass
+class ClassificationOptions:
+  """Options for classification processor.
+
+  Attributes:
+    display_names_locale: The locale to use for display names specified through
+      the TFLite Model Metadata.
+    max_results: The maximum number of top-scored classification results to
+      return.
+    score_threshold: Overrides the ones provided in the model metadata. Results
+      below this value are rejected.
+    category_name_allowlist: If non-empty, classifications whose class name is
+      not in this set will be filtered out. Duplicate or unknown class names are
+      ignored. Mutually exclusive with `class_name_denylist`.
```

Comment:
@khanhlvg I just noticed there were some minor typos in the docstrings so I've raised a small PR to fix them over here in this [PR](https://github.com/tensorflow/tflite-support/pull/821) since the current PR's already merged. PTAL.